source/includes/_clients.md

```diff
@@ -60,15 +69,25 @@ Update an existing client. Only supply values you want to update. The rest will
 use BrightLocal\Api;
 
 $api = new Api('<INSERT_API_KEY>', '<INSERT_API_SECRET>');
-$success = $api->put('v1/clients-and-locations/clients/', [
+$success = $api->put('v1/clients-and-locations/clients/', array(
     'name' => 'Le Bernardin',           
     'company-url' => 'http://www.example.com',
     'business-category-id' => 791
     
-]);
+));
 print_r($success);
 ```
 
+```shell
+curl -X PUT \
```

Comment:
This won't work. You need to also pass the API secret and expiry.

-d 'sig=\<INSERT_API_SIG\>' \
-d 'expires=\<INSERT_API_EXPIRES\>' \

Also indentation seems a little out on parameters. We're using spaces in files. Are you using tabs perhaps?